content/post/2025-04-08-hostless/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Fink hostless module
+date: 2025-04-08
+cardimage: 
+---
+
+Detecting the absence of something may seem challenging, how do you find what isn't there? This is the concept of ELEPHANT.
+<!--more-->
+
+![Hostless detection](assets/hostless-stamps.png)
```

Comment:
Thanks! there should be a folder `images` in the same folder than the post (`mkdir 2025--04-08-hostless/images`) with the image in it, and the path in the file should be:

```
![Hostless detection](images/hostless-stamps.png)
```